spec/models/job_spec.rb

```diff
@@ -316,7 +316,7 @@
       it "returns hash with job's attributes to use for syncronization with linked task" do
         expect(@job.attributes_for_task).to include(
           :status        => @job.status.try(:capitalize),
-          :state         => @job.state.try(:capitalize),
+          :state         => @job.state == "waiting_to_start" ? MiqTask::STATE_QUEUED : @job.state.try(:capitalize),
```

Comment:
This should be a specific value and not code.  That is, this test isn't being reused for multiple tests, so there should only be one possible value here.